tests/kw_remote_test.sh

```diff
@@ -164,6 +165,45 @@ function test_add_new_remote_multiple_entry_with_duplication()
   compare_array_values expected_result final_result_array "$LINENO"
 }
 
+function test_add_new_multiple_remotes_and_use_set_default_option()
+{
+  local output
+  local final_result_array
+
+  declare -a expected_result=(
+    '#kw-default=debian-machine'
+    'Host origin'
+    '  Hostname test-debian'
+    '  Port 3333'
+    '  User root'
+    'Host debian-machine'
+    '  Hostname test-debian'
+    '  Port 22'
+    '  User root'
+    'Host arch-machine'
+    '  Hostname la-debian'
+    '  Port 22'
+    '  User juca'
+  )
+
+  options_values['PARAMETERS']='origin root@test-debian:3333'
+  output=$(add_new_remote)
+
+  options_values['DEFAULT_REMOTE_USED']=1
+  options_values['DEFAULT_REMOTE']='debian-machine'
+  options_values['PARAMETERS']='debian-machine root@test-debian'
+  output=$(add_new_remote)
```

Comment:
Shouldn't there be an assertion here?

Reply:
We don't need any assertion because this test just populates the `remote.config` file.